package.json

```diff
@@ -164,6 +165,14 @@
     "webpack-cli": "^4.5.0",
     "yorkie": "^2.0.0"
   },
+  "peerDependencies": {
+    "jiti": "^1.21.0"
```

Comment:
Looks like this is in both `dependencies` and `peerDependencies` with different versions? I think it should just be `peerDependencies`?

Reply:
It's added as a dev dependency (not a prod dependency) and an optional peer dependency.